pallets/interest-accrual/src/lib.rs

```diff
@@ -417,16 +415,6 @@ pub mod pallet {
 			})
 		}
 
-		pub fn upgrade_to_v1() -> Weight {
```

Comment:
I think `remove_unused_rates` was also part of the migration and can be dropped.

Reply:
It seems unused, dropped it 👍

Reply:
@branan appreciate your confirmation here

Reply:
Yup, that was part of the migration (a nasty back-and-forth of data between interest-accrual and loans). Kill it with fire

Reply:
There is also a `reference_active_rates` in pallet-loans that can go bye-bye now, which was part of the same dance

Reply:
thanks @branan, done 👍